<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPreviewTestClass</class>
 <widget class="QMainWindow" name="QPreviewTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QPreviewTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="m_pWidget" native="true">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-1</y>
      <width>1171</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBGpuMode">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>760</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>790</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化布局：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnStopAll">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>630</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>停止所有预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>760</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API网关端口：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETAppkey">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>670</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnInit">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>730</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>传输协议：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETSecret">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>700</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>299</x>
      <y>700</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>主子码流类型：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETCameraIndex">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>670</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETIp">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>730</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>700</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>secret：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>755</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBTransMode">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>730</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>820</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>抓图存储路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBStreamType">
    <property name="geometry">
     <rect>
      <x>409</x>
      <y>700</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>730</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>API网关IP地址：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>670</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>appkey：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnPreview">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>开始预览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>760</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>是否启用GPU硬解：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETSnapDir">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>820</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="m_pBtnUninit">
    <property name="geometry">
     <rect>
      <x>729</x>
      <y>630</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>反初始化</string>
    </property>
   </widget>
   <widget class="QComboBox" name="m_pCBLayout">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>788</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="m_pEdtMsgTip">
    <property name="geometry">
     <rect>
      <x>589</x>
      <y>670</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_pLETPort">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>760</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>670</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监控点编号：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QPreviewTest.qrc"/>
 </resources>
 <connections/>
</ui>
